metadata-ingestion/src/datahub_provider/_plugin.py

```diff
@@ -302,8 +301,6 @@ def _patch_policy(settings):
 
 
 def _patch_datahub_policy():
-    print("Patching datahub policy")
```

Comment:
imo this one should still be logger.info - I believe this one only gets printed once (as opposed to per dag per task), so a higher log level is ok

Reply:
4e26016b2e0965fc390ae2760ad1f22eb78ef3a3